src/Ztm.Data.Entity.Postgres/TypeMappingSource.cs

```diff
@@ -0,0 +1,37 @@
+using Microsoft.EntityFrameworkCore.Storage;
+using NBitcoin;
+using Npgsql.EntityFrameworkCore.PostgreSQL.Infrastructure.Internal;
+using Npgsql.EntityFrameworkCore.PostgreSQL.Storage.Internal;
+using Ztm.Data.Entity.Postgres.Mapping;
+
+namespace Ztm.Data.Entity.Postgres
+{
+    public class TypeMappingSource : NpgsqlTypeMappingSource
```

Comment:
This class should be internal due to it not designed to use outside this assembly.